src/hooks/useStatus.js

```diff
@@ -0,0 +1,18 @@
+import useManifest from './useManifest'
+
+const determineFirstOnPage = (page, pageSize) => Math.max(page * pageSize, 0)
```

Comment:
I feel like this method should just add 1, rather than adding 1 to it everywhere it's used